nil/getting-started/essentials/creating-a-wallet.mdx

```diff
@@ -0,0 +1,56 @@
+# Creating a wallet
+
+This tutorial defines how wallets work in =nil; and outlines the different ways for creating a new wallet.
+
+## Definition
+
+In =nil; a wallet is any smart contract that is able to pay for interactions with real-life users.
+
+The traditional definition of a wallet in Ethereum mostly refers to externally-owned accounts (EOAs) that are secured via private keys and natively hold information about user's balances.
+
+Wallets in =nil; differ from this definition. A =nil; wallet is always a smart contract, and it must be deployed as such. The wallet handles all the usual functionalities (e.g., authenticating requests) but it can also include complex logic such as multi-sig operations.
```

Comment:
I think it is important to say here that since messages are used to trigger smart contract execution, in almost all cases an external message first goes into user's wallet, is authenticated, some value is attached to it and only then it is forwarded to the actual smart contract for execution.

This can be a big difference to how people are used to think about smart contract calls.